package.json

```diff
@@ -32,6 +33,11 @@
     "lerna": "^3.21.0",
     "lerna-changelog": "^1.0.1",
     "npm": "^6.14.5",
+    "ts-node": "^8.10.2",
     "typescript": "^3.9.2"
+  },
+  "dependencies": {
+    "@bubblewrap/cli": "^1.3.1",
```

Comment:
@andreban I'm not 100% on how our dependencies work, but should this be here? If so it's a bit weird that it wasn't here before.

Reply:
This is the root `package.json`. No need for this dependency here.